common/modal.js

```diff
@@ -58,6 +58,10 @@
         var exception = params.exception,
             reloadMessage = ' <p>  </p>';
 
+        if (ERMrest && exception instanceof ERMrest.NotFoundError && vm.params.showLogin) {
```

Comment:
I think this should be moved down to the if/else statement below. The conditional can be written like line 72 and the modifying the message like line 71.
```
} else if (ERMrest && exception instanceof ERMrest.NotFoundError) {
    if (vm.params.showLogin) vm.params.message = vm.params.message + messageMap.maybeNeedLogin;
} else if ...
```

Reply:
I didn't add it as `else if` case because [the logic in the `else` case](https://github.com/informatics-isi-edu/chaise/blob/e80396ead64ea82770bb3d38a4a205f45389c293/common/modal.js#L76,L82) still applies to this case.

Reply:
I could duplicate those lines in the the `else if` that I'm adding but that seemed wrong to me.

Reply:
Ahh, okay. That makes sense. The `if/else` is meant for conditionally setting the `clickActionMessage`. So maybe it would instead make sense to create an `if/else` statement (off of the one you added) that includes other changes to the message variable.